### DISCUSSION TITLE
View resulting curve in OUTPUTS

### DISCUSSION THREAD

**Discussion by fiechr** [EdgeTX/edgetx#4486] [Q&A]
As far as I understand, the resulting curve is only visible in the INPUTS page (ignoring the CURVES page for now). I like this feature, because even on a low resolution screen I instantly get an idea what different expo/diff and other functions like abs(x) or abs(f) do. Unfortunately, as far as I know, there is no way to display the resulting curves in the MIXES and OUTPUTS page (only the option to apply a custom curve).

Using the Channel Monitor or other screens, I can display a the numbers or a bar graph, but no curve.

Setting up a car, I set throttle CH2 in INPUTS to a diff value of 75 (to make it go backwards only 25% the speed) and then applied an expo value of 30 in the MIXES to the same channel to get more resolution in the lower range. I now wanted to find out, if these changes get applied on top of each other or if the expo setting simply replaces the curve coming from the INPUTS.

I wonder if there's a way to see the resulting curve not only in the INPUTS, but also in MIXES and OUTPUTS pages, respectively? For example by creating a Lua script? Or is there another way I did not realize, yet? Has this been done before? It also would be a nice way for beginners to visualize what all the different options, values and functions actually do.

**Comment by pfeerick** [comment]
Since you mention car, I guess you're talking about the MT12, and hence the black and white 128x64 user interface?

**Reply by fiechr** [reply]
Yeah, exactly!
